docker/Dockerfile.cpu

```diff
@@ -47,7 +47,7 @@ RUN python3 -m pip install -U 'pip<20' 'setuptools<50' && python3 -m pip install
     minio==4.0.5 \
     rl-coach-slim==1.0.0 \
     PyYAML==3.13 \
-    rospkg>=1.1 \
+    rospkg\>=1.1 \
```

Comment:
Can we please use single quotes, not escape character - just to be coherent with the rest of the file.

Reply:
Absolutely!  You are welcome to use whatever you like.

Reply:
If you change it on your end then the commits belong to you. :)

Reply:
Right... as if I cared...

BUT, if I were to make changes beyond the minimal fixes proposed here,
I'd probably want to revamp the 4 largely identical `Dockerfile`:s and move
the parts that actually differ, which appear to be the base image and variations
in package lists, into build arguments (similarly to  `TENSORFLOW_VER`).

For instance, apart from white-space, commented lines, and the ordering of
the ROS packages, the only difference between `Dockerfile.?pu` is the base image.

Similarly, the differences betwen GL and non-GL also seem minor, albeit there
I'm unsure I grok why _some_ of the dependencies are included in one, but
not the other...   or why certain packages (mostly UI dependencies) happen to
get installed in the Gazebo block in the non-GL vs up-front in the GL variants.
Then, there is `TZ` being set in non-GL only.  Dependencies are _sometimes_
explicitly listed, sometimes not.

Finally, the 2 sample Dockerfiles make very little sense (the one with 2 `CMD`
statements is really a no-op; the other one seems like a bad idea for size :smirk:).
In either case, their sensible reincarnations could be moved to `examples`...

If I were to touch anything, I'd also _have to_ replace all `gawk`isms by corresponding
`bash`isms in the different scripts, since the shell can handle pretty much everything
`[g]awk` is used for in this repo.

Let me know if this is of interest; and if so please help to clarify the issues
I mentioned above.

BTW, where does the `bundle` come from?  Is that _officially_ released?
I cannot seem to find its public source.  The S3 URI in the utility script does
NOT appear to be publicly accessible... (or, I have really bad luck).
https://github.com/aws-deepracer-community/deepracer-simapp/blob/8bfce605e895e5ad805756be959e9dacb943000c/utils/extract-and-diff.sh#L3

Reply:
@andraxin - happy to take on help from anyone interested in tuning the scripts and/or assist in maintaining the repository in general.

Some points:
* The bundle is available and is made public by AWS to enable people to alter it. The primary idea is that it is then run from a Jupyter notebook.
* The source zip is an entire root FS; and contains a already BUILT ROS `install` directory; we are extracting the source out of it, which is rather dirty. The real source is not available as a 'clean' repository.
* The `bundle/` is having a set of alterations which the community has added over time. Primary change is to allow a local S3 (using s3 endpoint URL), but there are other tweaks / small additional features added as well. 
* In general changes are made using as little edits as possible, as the 'apply diff' approach works best if the differences are small.